src/pages/RFOX/components/Overview/Overview.tsx

```diff
@@ -1,33 +1,27 @@
 import { Card, CardBody, CardHeader, Divider } from '@chakra-ui/react'
-import { fromAccountId } from '@shapeshiftoss/caip'
 import { useMemo } from 'react'
 
 import { StakingInfo } from './StakingInfo'
 import { Stats } from './Stats'
 
-import { useRFOXContext } from '@/pages/RFOX/hooks/useRfoxContext'
+import { supportedStakingAssetIds, useRFOXContext } from '@/pages/RFOX/hooks/useRfoxContext'
```

Comment:
Now living at top-level export - no need for this to be part of context and require `useRfoxContext()` (e.g would cause crashes in places that do *not* have access to the context)